proposed/container.md

```diff
@@ -97,9 +97,9 @@ into the same container (or another container) instead of the delegate container
 The interfaces and classes described as well as relevant exception are provided as part of the
 [psr/container](https://packagist.org/packages/psr/container) package. (still to-be-created)
```

Comment:
`(still to-be-created)` should be removed too as the package already exists, right?